ui/js/service/http.service.ts

```diff
@@ -40,7 +40,7 @@ export class HTTPService {
     return fromFetch(endpoint, {
       method: "POST",
       headers: {
-        Authorization: "Bearer " + localStorage.getItem("jwt"),
+        Authorization: this.authHeader(options)
```

Comment:
This is great. Thank you. However, I see that you only did it in the `post` method. I think we can reuse this in the `get, put and delete` method.

Reply:
Yeah I wasn't sure we needed them there because the login (which is post) would be the only one where the front end is sending username/password.  No harm in doing it everywhere though, especially if we eventually add more strategies.